src/wasm.h

```diff
@@ -59,6 +59,7 @@
 #include "pretty_printing.h"
 #include "support/bits.h"
 #include "support/utilities.h"
+#include "support/metrics.h"
```

Comment:
No, we only need it if it's used. Since I broke up the patch in two PRs, this and #208 which actually uses it, I at least wanted to make sure it's part of the build.